ui/vue.config.js

```diff
@@ -8,7 +8,7 @@ const vueSrc = "./src";
 
 module.exports = defineConfig({
   transpileDependencies: true,
-  lintOnSave: true,
+
```

Comment:
lintOnSave value is either true or false, and the change will only affect the code if the code editor is set to follow eslint... and I dont think all of us have set our code editor to follow eslint, so I deleted it because its unnecessary

Reply:
eh no, it can be set to warning... hmmm 
either way my stance stays, because I said I dont think all of us have set their code editor to follow eslint.. should I just bring this config back?

https://cli.vuejs.org/config/#lintonsave

Reply:
Hmm, you can put it back and set it to warning. I will try to configure eslint as well, if possible.

Reply:
Alright, I brought it back.